app/controllers/organized/base_controller.rb

```diff
@@ -6,14 +6,6 @@ class BaseController < ApplicationController
 
     layout "application_revised"
 
-    def root
```

Comment:
I think it makes sense to move this out of base_controller, because all the other organized controllers inherit from this controller, and we probably don't want them all to have a `root` method?